src/Search/Search/Search.csproj

```diff
@@ -11,7 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Search" Version="2.0.1" />
+    <PackageReference Include="Microsoft.Azure.Management.Search" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Azure.Search" Version="9.0.0" />
```

Comment:
We shouldn't need references to the `Microsoft.Azure.Search` packages here, only `Microsoft.Azure.Management.Search`.

Reply:
Yes, I also think only mgmt. plane SDK should be enough. If not, we should add missing pieces into it.